### ISSUE TITLE
VideoIn (CL Eye) throws CLMulticam.dll missing

### ISSUE THREAD

**Issue by tekcor** [elliotwoods/VVVV.Packs.Image#59]
Hi,

Status pin from VideoIn (CL Eye) and ListDevices (CL Eye) says the CLMulticam.dll would be missing.

Unable to load DLL 'CLEyeMulticam.dll': Invalid access to memory location. (Exception from HRESULT: 0x800703E6)

The dll is in the folder. So maybe it is some path linking issue?

Happens to me only when one or more PS3 Eye cams are connected and activated.

Without it says 'OK'

Would realy love to get this Multicam thing to work!

e: i am running b32.1_x86 with alg's latest pack from the contribution

**Comment by elliotwoods** [comment]
Hey Tektor!
Sorry for the slow reply here. I realised my email notifications were off for a while.

From your descrption it sounds like maybe you can get one camera working but the second one doesn't? Or could you clarify what you mean by: 

> Happens to me only when one or more PS3 Eye cams are connected and activated.

**Comment by timpernagel** [comment]
Hey,

running into nearly the same issue, but with an other additional error message:

"Unable to load DLL 'CLEyeMulticam.dll': A dynamic link library (DLL) initialization routine failed. (Exception from HRESULT: 0x8007045A)"

The ode worked for a while, but now I've got constantly this error message.

**Comment by timpernagel** [comment]
My workaround is to use a bat-file and start the video-patch first and than start the rest of the project. Seems, the bug only happens if some node is trying to access the texture when it is not already initialized.

**Comment by sunep** [comment]
Could it have something to with with code laboratories having dropped their free drivers and only having drivers you need to pay for?

**Comment by elliotwoods** [comment]
yes i presume you need to have drivers which are compatible with this build (which is pretty old)
if they’ve changed their API too much then it won’t be compatible

**Comment by ggml** [comment]
the current driver is identical to the free one before (5.3.0.0341)
for me it was a sudden halt in a configuration that worked for weeks, now it wont work at all:
b337, pack from september 2013
it will crash for every videoin activation

**Comment by timpernagel** [comment]
Does even the Help-Patch crashes? This works fine for me.

**Comment by sebllll** [comment]
i guess the problem is, that the pack uses a precompiled CLEyeMulticam.dll. and that dll probably was compiled with the old and free driver thats no longer available

**Comment by ggml** [comment]
old and new driver are the same, v.5.3.0.0341

**Comment by sebllll** [comment]
maybe @smakhtin can say a bit here. he seems to be the author of that dll. see https://www.nuget.org/packages/VVVV.CLEyeMulticam/1.2.0.1

**Comment by ggml** [comment]
here is a report of what is going on in my previously working setup
http://vvvv.org/forum/strange-experience-with-cleye#comment-190992
